### PR TITLE
Tunnel status code

### PR DESCRIPTION
**Pull request by mary-dcouto**
Adding Code field as UI depends on this status code. Changed Status field to IdFileStatus, as old config file already has a Status field which is of string data type

### CODE REVIEW

File: lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h [hunk 1]
```diff
@@ -307,6 +308,14 @@ struct add_identity_request_s {
     void *cmd_ctx;
 };
 
+#ifndef IPC_SUCCESS
```
Comment (scareything) [inline lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h:311]: why protect these in `#ifdef`s? do we want to allow user-supplied values of IPC_SUCCESS / IPC_ERROR?

File: lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h [hunk 2]
```diff
@@ -307,6 +308,14 @@ struct add_identity_request_s {
...
+#define IPC_SUCCESS 0
+#endif
+#ifndef IPC_ERROR
+#define IPC_ERROR 500
```
Comment (scareything) [inline lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h:315]: 500 feels like these are maybe sorta trying to be like http status codes? If so should IPC_SUCCESS be 200?

Comment (mary-dcouto) [reply lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h:315]: Currently, there are different error codes defined for different types of error. These are not similar to http error codes. However, UI is checking for only 0 (success) and 500(error) error codes, so I copied only those 2 error codes in to t-sdk.

File: lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h (hunk 1 above)
Comment (mary-dcouto) [reply lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h:311]: these are constant fields. I will remove the ifdef

**Pull request merged**